rollup/plugins/bundlePlugins.js

```diff
@@ -63,6 +63,14 @@ export function makeIsDebugBuildPlugin(includeDebugging) {
   });
 }
 
+export function makeIsCDNBundlePlugin(isCDNBundle) {
+  return replace({
+    values: {
+      __SENTRY_CDN_BUNDLE__: isCDNBundle,
```

Comment:
IIRC the replace plugin wants us to set some `preventAssignment` option. I would set it to false here.